src/deployments/inputs/HorizonPhaseOneUpdate.sol

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.0;
+
+import {IAaveV3ConfigEngine as IEngine} from '../../contracts/extensions/v3-config-engine/IAaveV3ConfigEngine.sol';
+import {EngineFlags} from '../../contracts/extensions/v3-config-engine/EngineFlags.sol';
+import {AaveV3Payload} from '../../contracts/extensions/v3-config-engine/AaveV3Payload.sol';
+
+contract HorizonPhaseOneUpdate is AaveV3Payload {
+  address public immutable GHO_ADDRESS;
+  address public immutable USDC_ADDRESS;
+  address public immutable RLUSD_ADDRESS;
+  address public immutable USTB_ADDRESS;
+  address public immutable USCC_ADDRESS;
+  address public immutable USYC_ADDRESS;
+  address public immutable JTRSY_ADDRESS;
+  address public immutable JAAA_ADDRESS;
```

Comment:
we should make these constants, no need to be immutable since hardcoded in the constructor